PF/Assignment1/VehicleHelper.java

```diff
@@ -0,0 +1,20 @@
+//Class to create vehicle
+public class VehicleHelper {
```

Comment:
ACTION: Do we actually required this class now? can't we put "createVehicle" method in "Vehicle" class?